### ISSUE TITLE
Can not copy settings to system profile [fresh install]

### ISSUE THREAD

**Issue by feerrenrut** [nvaccess/nvda#9448]
### Steps to reproduce:
1. Uninstall NVDA
1. Remove user profile from `%appdata%/nvda`
1. Remove system configuration profile from `C:\Program Files (x86)\NVDA\systemConfig`
1. Install NVDA
1. Attempt to "Use currently saved settings on the logon and other secure screens (requires administrator privileges)" (button on the general settings panel [`NVDA+control+g`])

### Actual behavior:
An error sound is heard, and the following error in the log:

```
ERROR - unhandled exception (18:46:34.341):
Traceback (most recent call last):
  File "gui\settingsDialogs.pyc", line 730, in onCopySettings
WindowsError: [Error 3] The system cannot find the path specified: u'C:\\Users\\Reef\\AppData\\Roaming\\nvda\\appModules\\*.*'
```

### Expected behavior:
The configuration should be copied to the system profile.

#### NVDA version:
2019.1

#### Windows version:
Windows 10

#### Name and version of other software in use when reproducing the issue:
None

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA?
2018.4 - The issue does not occur

**Comment by josephsl** [comment]
Hi, perhaps a leftover from pre-scratchpad commits? Thanks.

**Comment by feerrenrut** [comment]
Thanks Joseph. Yep, I have a fix on it's way.

**Comment by ehollig** [comment]
Closing as fixed in #9449
